### ISSUE TITLE
Does not start on macOS Big Sur 11.2.3

### ISSUE THREAD

**Issue by Chibibowa** [MatthewPierson/Ramiel#22]
Here is the crash report I got:

Process:               Ramiel [842]
Path:                  /Applications/Ramiel.app/Contents/MacOS/Ramiel
Identifier:            com.moski.Ramiel
Version:               1.0.3 (1)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Ramiel [842]
User ID:               501

Date/Time:             2021-03-22 14:39:32.883 +0100
OS Version:            macOS 11.2.3 (20D91)
Report Version:        12
Anonymous UUID:        DCADEBF6-705B-8A54-F2CB-9DA078C3EE26

Time Awake Since Boot: 330 seconds

System Integrity Protection: disabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  dyld: Using shared cache: 52762DF1-0E55-3F8B-9588-8169E6D0EB95
Library not loaded: /usr/local/opt/openssl@1.1/lib/libcrypto.1.1.dylib
  Referenced from: /Applications/Ramiel.app/Contents/MacOS/Ramiel
  Reason: image not found

Binary Images:
       0x102adb000 -        0x102b4afff +com.moski.Ramiel (1.0.3 - 1) <8DC8A987-9E39-3676-8966-79DC8E4D1F92> /Applications/Ramiel.app/Contents/MacOS/Ramiel
       0x102b6a000 -        0x102b91fff +org.cocoapods.AFNetworking (2.7.0 - 1) <B8D00AB1-29D0-3DCA-B2C4-6A8E96C1808B> /Applications/Ramiel.app/Contents/Frameworks/AFNetworking.framework/Versions/A/AFNetworking
       0x102bea000 -        0x102c09fff +org.cocoapods.SSZipArchive (2.2.3 - 1) <C16C6A31-E454-3096-AE20-DBD4D8AE94BA> /Applications/Ramiel.app/Contents/Frameworks/SSZipArchive.framework/Versions/A/SSZipArchive
       0x102c33000 -        0x102c39fff +libusbmuxd-2.0.6.dylib (0) <ABA49CB9-D051-3325-B415-18CE7EE402A6> /Applications/Ramiel.app/Contents/Frameworks/libusbmuxd-2.0.6.dylib
       0x102c46000 -        0x102c55fff +libusb-1.0.0.dylib (0) <58A99A32-16D5-39AD-8F6F-160E1B8BEEF6> /Applications/Ramiel.app/Contents/Frameworks/libusb-1.0.0.dylib
       0x10a732000 -        0x10a7cdfff  dyld (832.7.3) <0D4EA85F-7E30-338B-9215-314A5A5539B6> /usr/lib/dyld
    0x7fff204fa000 -     0x7fff20995fff  com.apple.CoreFoundation (6.9 - 1774.101) <2A0E160E-9EE6-3B23-8832-6979A16EC250> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
    0x7fff20fbb000 -     0x7fff21038fff  com.apple.SystemConfiguration (1.20 - 1.20) <3518EA0E-C32D-32CC-81B9-0F3C83B6430C> /System/Library/Frameworks/SystemConfiguration.framework/Versions/A/SystemConfiguration
    0x7fff22334000 -     0x7fff22685fff  com.apple.security (7.0 - 59754.80.3) <B377D7C7-EDB6-3737-B492-E9872F4C6469> /System/Library/Frameworks/Security.framework/Versions/A/Security
    0x7fff2a5bd000 -     0x7fff2a5cffff  libz.1.dylib (76) <6E2BD7A3-DC55-3183-BBF7-3AC367BC1834> /usr/lib/libz.1.dylib
    0x7fff2a6c4000 -     0x7fff2a7b4fff  libiconv.2.dylib (59) <AD10ECF4-E137-3152-9612-7EC548D919E8> /usr/lib/libiconv.2.dylib
    0x7fff2a8fd000 -     0x7fff2a915fff  libcompression.dylib (96.40.6) <9C42DB2C-0A6D-38CE-BB8D-899A9B34B695> /usr/lib/libcompression.dylib
    0x7fff30074000 -     0x7fff30074fff  com.apple.CoreServices (1122.11 - 1122.11) <E59859C6-7221-3324-BB58-F910B2199959> /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices

Model: iMacPro1,1, BootROM 1554.40.41.0.0, 8 processors, 8-Core Intel Core i7, 3,8 GHz, 16 GB, SMC 
Graphics: AMD Radeon VII, AMD Radeon VII, spdisplays_pcie_device, 16 GB
Memory Module: BANK 0/ChannelA-DIMM1, 4 GB, DDR4, 3000 MHz, Corsair, CMK16GX4M4B3000C15
Memory Module: BANK 1/ChannelA-DIMM2, 4 GB, DDR4, 3000 MHz, Corsair, CMK16GX4M4B3000C15
Memory Module: BANK 2/ChannelB-DIMM1, 4 GB, DDR4, 3000 MHz, Corsair, CMK16GX4M4B3000C15
Memory Module: BANK 3/ChannelB-DIMM2, 4 GB, DDR4, 3000 MHz, Corsair, CMK16GX4M4B3000C15
AirPort: spairport_wireless_card_type_third_party
Bluetooth: Version 8.0.3d9, 3 services, 27 devices, 1 incoming serial ports
Network Service: Ethernet 2, Ethernet, en1
Network Service: Wi-Fi, AirPort, en2
Serial ATA Device: WDC WD1003FZEX-00MK2A0, 1 TB
Serial ATA Device: WDC WD10EZEX-00WN4A0, 1 TB
USB Device: USB 3.1 Bus
USB Device: USB2.0 Hub
USB Device: BCM920702 Bluetooth 4.0
USB Device: iPhone
USB Device: AURA LED Controller
USB Device: ROG Gladius II
USB Device: Razer Kiyo
USB Device: USB2.0 Hub
USB Device: CORSAIR Lighting Node CORE
USB Device: H100i Platinum
USB Device: General Purpose USB Hub
USB Device: Arctis Pro Wireless
USB Device: Arctis Pro Wireless
USB Device: Smart Card Reader Interface
USB Device: Corsair Gaming K95 RGB PLATINUM Keyboard
Thunderbolt Bus:

**Comment by sthomsonpl** [comment]
`brew` install `openssl`

**Comment by Chibibowa** [comment]
The matter is solved. I figured it out after all, but couldn’t understand why because the first version you pushed worked and after checking your updated version, all the dylibs were included so I was quite confused. 

You might want to add this step to your main page. Alongside some tutorials to set the nonce for example, I tried and it failed. Didn’t understand why. Device was an 8 Plus without passcode freshly restored. Unless Ramiel isn’t fully optimized for Big Sur. 

Lastly, maybe caused by Big Sur; version 1.0.3 takes a long time to launch. It keeps bouncing for an easy 30 seconds before popping up. This behavior persisted after a reboot. I didn’t have this problem with your first version. 

Let me know if I can do anything to help your research.
